src/pin_layout.rs

```diff
@@ -160,16 +160,20 @@ fn get_pin_widget(
 
             if is_left {
                 // TODO Row::new().push(waveform(16.0, 256.0, pin_state)).push(
-                Row::new().push(led(16.0, pin_state)).push(
-                    pick_list(sub_options, *pull, move |selected_pull| {
-                        Message::PinFunctionSelected(
-                            board_pin_number,
-                            bcm_pin_number.unwrap(),
-                            Input(Some(selected_pull)),
-                        )
-                    })
-                    .placeholder("Select Pullup"),
-                )
+                Row::new()
+                    .push(led(16.0, pin_state))
```

Comment:
There's a fixed size for led (16.0) there that we could review in the future?

Reply:
sure